CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+<a name="v0.9.11"></a>
+# [v0.9.11](https://github.com/qri-io/qri/compare/v0.9.10...v0.9.11) (2020-08-10)
+
+This patch release addresses a critical error in `qri setup`, and removes overly-verbose output when running `qri connect`.
+
+### Bug Fixes
+
+* **lib:** don't panic when resolving without a registry ([2708801](https://github.com/qri-io/qri/commit/2708801))
+* **p2p:** some clean up around `qri peers connect` and `upgradeToQriConnection` ([#1489](https://github.com/qri-io/qri/issues/1489)) ([b7bb076](https://github.com/qri-io/qri/commit/b7bb076))
+* **setup:** Fix prompt, add a test for --anonymous ([3b2c58a](https://github.com/qri-io/qri/commit/3b2c58a))
+* **setup:** Fix setup, add many unit tests ([b88d084](https://github.com/qri-io/qri/commit/b88d084))
+
```

Comment:
Also prevents segfault in deepdiff when running qri save.

Reply:
I think we should handle this in the future by tagging dependency version bumps with `fix`

Reply:
for now, it's in the yet-to-be-made release notes of deepdiff (when we cut a deep diff release)

Reply:
Sounds fine. Just wondering in this case because it directly affects functionality.

Ah, I see I could have used `fix` instead of `chore`, that makes sense.